dependencies.yaml

```
@@ -78,6 +84,18 @@ dependencies:
       - output_types: [conda, requirements]
         packages:
           - gcovr>=5.0
+  docs_builds:
+    common:
+      - output_types: [conda]
+        packages:
+          - doxygen=1.8.20
```

We probably require `graphviz` for the `dot` command. There were some errors @ajschmidt8 reported in the PR comment thread related to `dot`. That tool generates the inheritance graphs shown here: https://docs.rapids.ai/api/librmm/nightly/inherits.html

```suggestion
          - doxygen=1.8.20
          - graphviz
```

---

Yeah just fixed that